src/main/scala/firrtl/annotations/MemoryInitAnnotation.scala

```diff
@@ -52,3 +52,7 @@ case class MemoryFileInlineAnnotation(
   override def initValue:    MemoryInitValue = MemoryFileInlineInit(filename, hexOrBinary)
   override def isRandomInit: Boolean = false
 }
+
+/** Defines the memory initialization target, inside or outside the `ifndef SYNTHESIS` block */
```

Comment:
You should add a scala doc comment to each annotation describing what each one individually does.

Reply:
Added!